### DISCUSSION TITLE
I get 404 error when trying access some documentation links

### DISCUSSION THREAD

**Discussion by Camriver3** [google/meridian#261] [Q&A]
Getting 404 Not found for:

- migration guide: https://developers.google.com/meridian/docs/migrate
- meridian getting started notebook: https://developers.google.com/meridian/notebook/meridian-getting-started (we can access using GitHub)
- User guide: https://developers.google.com/meridian/docs/user-guide/installing
- Pre-modeling: https://developers.google.com/meridian/docs/user-guide/collect-data

**Comment by vivekbeniwal** [comment]
Hi @Camriver3 

You need to sign in to https://developers.google.com/meridian to access these links through your google linked account.

Can you reach out to meridian-comms@google.com with subject: "[Access Issue]" and someone will help you out with this.

Regards
Vivek
